libafl/src/feedbacks/map.rs

```diff
@@ -318,6 +318,8 @@ where
 {
     /// Contains information about untouched entries
     pub history_map: Vec<T>,
+    /// Tells us how many non-zero entries there are in `history_map`
```

Comment:
this should be `non-default` instead of `non-zero`

because sometimes the default value might not be 0

Reply:
true